backend/pkg/api/router.go

```diff
@@ -94,8 +94,11 @@ func addRoutes(hs *handlers.HandlerService, publicRouter, internalRouter *mux.Ro
 		{http.MethodGet, "/product-summary", nil, hs.InternalGetProductSummary},
 
 		{http.MethodPost, "/users", nil, hs.InternalPostUsers},
-
 		{http.MethodGet, "/users/me", nil, hs.InternalGetUserInfo},
+		//{http.MethodDelete, "/users/me", nil, hs.InternalDeleteUser},
+		//{http.MethodPut, "/users/me/email", nil, hs.InternalPutUserEmail},
+		//{http.MethodPut, "/users/me/password", nil, hs.InternalPutUserPassword},
+		//{http.MethodPut, "/users/me/subscriptions", nil, hs.InternalPutUserSubscriptions},
```

Comment:
Changing the subscription is not really part of this BIDS (that will be done on the pricing page I guess), other endpoints look fine